### PR TITLE
HBW-211 Notify user about new assigned tasks

### PR DESCRIPTION
**Pull request by TimAle**
Снимок экрана 2019-11-15 в 15 13 28

### CODE REVIEW

File: hbw/config/locales/en.yml [hunk 1]
```diff
@@ -17,6 +17,9 @@ en:
         file_list_field_required: Field of "file_list" type with name "homsOrderDataFileList" required
         field_is_required: Field is required
 
+      notifications:
+        new_assigned_task: "%{count} new assigned task(s): %{task_names}"
```
Comment (sugarfree1) [inline hbw/config/locales/en.yml]: cmon, lets create 2 translations (singular/plural)

File: hbw/app/javascript/packs/hbw/components/claiming/menu_button.jsx [hunk 2]
```diff
@@ -53,15 +55,18 @@ class HBWClaimingMenuButton extends Component {
...
+            <HydraIcon />
+            <span className="claiming-count">
+              {myTasksCount}/{unassignedTasksCount}
+            </span>
+          </div>
+          {isOpen && <PopUp perPage={10} env={this.props.env} count={count} />}
         </div>
-        {isOpen && <PopUp perPage={10} env={this.props.env} count={count} />}
-      </div>
+        <ClaimingNotifier perPage={400} env={env} />
```
Comment (sugarfree1) [inline hbw/app/javascript/packs/hbw/components/claiming/menu_button.jsx]: i didn't get where does this option come from? 
perPage

Comment (TimAle) [reply hbw/app/javascript/packs/hbw/components/claiming/menu_button.jsx]: this come from HOC wrapper `withClaimingTasks`

**Comment by TimAle** [comment]
Снимок экрана 2019-11-19 в 10 44 39

**Pull request merged**